### ISSUE TITLE
Replace @uiw/react-md-editor with a different Markdown editor

### ISSUE THREAD

**Issue by fgatti675** [Camberi/firecms#417]
At this moment, this is the only dependency that requires additional setup to run with Next.js and SSR
See https://github.com/Camberi/firecms/issues/415#issuecomment-1299336136
This would allow developers to use next.js out of the box

**Comment by KULTI1995** [comment]
bump :)

**Comment by fgatti675** [comment]
This has been implemented in the last 2.0.0 version.
FireCMS can be now used with next.js, at least with SSR switched off for now.
```
const Admin = dynamic(() => import("../your_firecms_app"), { ssr: false });
```
